xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -159,12 +153,10 @@ The `CurrentDepth` property tracks the recursive depth of the nested objects / a
         <Parameter Name="isFinalBlock" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="isFinalBlock"><see langword="true" /> to indicate no more data will be written; otherwise, <see langword="false" />. This is used to validate
-            that the JSON written so far is structurally valid if no more data is to follow.</param>
+        <param name="isFinalBlock"><see langword="true" /> to indicate no more data will be written; otherwise, <see langword="false" />. This is used to validate that the JSON written so far is structurally valid if no more data is to follow.</param>
```

Comment:
It looks like almost all of these changes are whitespace/formatting changes. Looking at a few places, the content remained unchanged.

Do we want to make this formatting change? Or should we keep it as it was (to make diff/review easier)?

Reply:
If it's only two spaces, I don't think we should worry. @mairaw can confirm if I'm wrong.
@ahsonkhan  if you feel like something needs to be changed, feel free to provide a code change suggestion so i can quickly merge it.

Reply:
There's a really cool feature in GitHub for code suggestions (and I've been learning this week that not everyone is aware it exists):
When you reply, the first icon on the top bar is the "Insert a suggestion" option (ctrl+g), which will allow you to suggest a code change by providing a diff that I can quickly merge.

Reply:
It's not too big of a deal so I wouldn't worry about it (although I do try to clean up when I see them because I'm picky 😄).

Yeah, I love the suggestions feature!

Reply:
Yep, me too. I am aware of it, thanks for sharing the details though :)

My concern was just that it makes reviewing such a diff difficult since I have to then double check that the content is identical (and only review what changed). Since the change is **ONLY** white space, ideally, as a reviewer, I can just ignore this file all together rather than trying to decipher what actually changed amongst the white-space changes. Is that the case here or has the text itself changed in some places?

Also, whenever there are such cosmetic changes caused by one tool, inevitably, a different generation tool reverts it. That perpetuates the cycle of noisy PRs. I don't have an actual preference on how the formatting should/needs to be stored here. I will gladly take whatever spacing/formatting works :)

Reply:
The text was changed everywhere where it used to say "To be added".

Double spaces came straight from the triple slash comments.

The modified endlines were removed manually by me because there were several sentences that were being cut in half.

Please don't ignore the file all together. I don't think the review is that difficult. Yes, it's a bit inconvenient that some text didn't change except for spaces/endlines, but it will only take you a few minutes to verify if the description for the API is correct. Hopefully, this will be the only time we will have to do such a big review.